### ISSUE TITLE
[Feature Request] Allow sorting of "Repeat Panels"

### ISSUE THREAD

**Issue by drewboswell** [grafana/grafana#5171]
It would be useful to be able to sort "repeat panel"s, i.e.

I'm not sure how this could be done, 

# Version information
Grafana-server: 3.0.2
Worldping version: 1.0.7
Server arch: Linux XEN x86_64 GNU/Linux

**Comment by mtanda** [comment]
https://github.com/grafana/grafana/issues/2712
This is not same feature, but this feature also checks metric values.
The purpose of feature is similar.

**Comment by mtanda** [comment]
If using Prometheus, this might be possible.
Prometheus datasource plugin has "query_result()" query, we can filter low cpu time metrics, and get server id list, which cpu time is higher than the threshold.
Moreover, we can sort the result by cpu time.

If https://github.com/grafana/grafana/pull/5129 is merged, and set "Without sort", we can display the graph panels, sorted by cpu time.

**Comment by gealtrag** [comment]
+1. Much needed feature.

**Comment by northtree** [comment]
@torkelo Could I know when this feature will be supported? I would like to hide some panels/rows based on metric/query. Thanks.

**Comment by torkelo** [comment]
@northtree no, this is an open feature request that is not in any milestone or roadmap

**Comment by AdamCavaliere** [comment]
Just adding a +1 here!

**Comment by ahcub** [comment]
+1

**Comment by shivam-bidgely** [comment]
+1

**Comment by n3k232** [comment]
It would be good to hide Empty Panels in Repeating Rows.
Need this Feature

**Comment by bavarianbidi** [comment]
+1

**Comment by tblok** [comment]
+1

**Comment by piaoyu** [comment]
+1

**Comment by jirkace** [comment]
+1

**Comment by justinrush** [comment]
+10000

**Comment by gorbi2001** [comment]
+1

**Comment by jcoconnor** [comment]
+50000

**Comment by mcsage** [comment]
+1

**Comment by Berkmann18** [comment]
+1

**Comment by RichardWright** [comment]
+1

**Comment by amir-am** [comment]
+1

**Comment by dengruilong** [comment]
nice feature, looking forward its release

**Comment by yuraxdrumz** [comment]
+1

**Comment by veronikaKochugova** [comment]
+1

**Comment by K0nne** [comment]
+1

**Comment by bluefishforsale** [comment]
+1

**Comment by x3nb63** [comment]
+1^666

**Comment by ABystroff** [comment]
+1

**Comment by petlik90** [comment]
+1

**Comment by xlw712** [comment]
+10086

**Comment by ubikry** [comment]
++

**Comment by chaoukibenfradj** [comment]
+1

**Comment by pylisboa633** [comment]
???? where this amazin feature?

**Comment by rfsburger** [comment]
+1

**Comment by mdvictor** [comment]
We are considering implementing conditional rendering in the future and this specific use case might be included as well. At this point though, we cannot offer a specific timeline.

**Comment by jounaidr** [comment]
+1

**Comment by ghost** [comment]
+1
